examples/datadogslo/metric-example.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogSLO
+metadata:
+  name: example-slo
+  namespace: system 
+spec:
+  name: example-slo
+  description: "This is an example SLO from datadog-operator"
+  query:
+    denominator: "sum:requests.total{service:example,env:prod}.as_count()"
```

Comment:
What if we use `numerator: "sum:universal.http.server.hits{service:example, env:prod, !http.status_class:5xx}.as_count()"` and `denominator: sum:universal.http.server.hits{service:example, env:prod}.as_count()"` instead of `requests.total`  and `request.success` as the default metrics? It's a bit more complicated, but if they have universal service monitoring, at least that stands a chance of working when they replace it with their service. If they don't have universal service monitoring, then at least we're advertising a Datadog product. Not a big deal to keep these defaults, though.

Reply:
I think we could benefit from having another example or two - what do you think about adding your query suggestion in another file `metric-usm-example.yaml` ?

Reply:
@ddjamesfrullo  I added another metric example using the query you suggested, as well as a monitor example.